src/mapping/query/functions.rs

```diff
@@ -32,12 +119,16 @@ impl Function for NotFn {
 #[derive(Debug)]
 pub(in crate::mapping) struct ToStringFn {
     query: Box<dyn Function>,
-    default: Option<Value>,
+    default: Option<Box<dyn Function>>,
 }
 
 impl ToStringFn {
+    #[cfg(test)]
```

Comment:
Production code now uses `TryFrom<ArgumentList>`, but I've kept the `new` functions around, as it makes testing much easier (i.e. way less casting to `Literal` and `Box<dyn Function>`).